### PR TITLE
Add Grouparoo Code of Conduct & Contribution Guide

### PR DESCRIPTION
**Pull request by evantahler**
Generated with help from @Github.

### CODE REVIEW

**Comment by bleonard** [comment]
👍

**Review by bleonard — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+### Contributions
+
+Before you start, please join us on slack??? to discuss your plans for the new feature with us. As we don't yet publicly share our roadmap, we want to be sure that you won't be duplicating work, and that we will eventually merge in your work. Failing that, members of Grouparoo Inc will endeavor to quickly comment on an open Issue or Pull Request that will not be able to be merged.
+
+At this time, Grouparoo Inc. is focused on expanding and enhancing our core features, in addition to developing our paid plugins. With that in mind, there are some areas of the codebase that may not be stable enough to accept Pull Requests, or areas of the product that are undergoing significant changes. There are also categories of work which fall outside the scope of of the core product which should be moved to a plugin. Similarly, Pull Requests that drastically change the functionality of an existing section of the product are unlikely to be merged, as they likely conflict with our roadmap.
```
Comment (bleonard) [inline .github/CONTRIBUTING.md:29]: I think "check in with us on slack, first, if you have an idea" at the end of this would be good.

**Review by bleonard — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+
+The best place to contribute to Grouparoo is by adding a new connection to add a new `Source` or `Destination` via a [plugin](https://github.com/grouparoo/grouparoo/tree/master/plugins/%40grouparoo).
+
+### Ownership
+
+By contributing to Grouparoo, you agree that you have granted ownership of the work to Grouparoo Inc, to be maintained and distributed in accordance with our [open-source License (Mozilla 2.0)](https://github.com/grouparoo/grouparoo/blob/master/LICENSE.txt).
+
+### Testing
+
+Grouparoo is a large project with lost of different plugins and tools. We don't expect you to know about everything, so that's why we have a robust test suite. This allows us to ensure that no matter who makes a change, Grouparoo will continue to work the way it is supposed to.
```
Comment (andyjih) [inline .github/CONTRIBUTING.md]: "lost" -> "lots"

**Review by andyjih — CHANGES_REQUESTED** [review]
Looks great! Just a typo :)

**Review by andyjih — APPROVED** [review]

**Pull request merged**